communication/vehicleconnections/mavsdkvehicleconnection.cpp

```diff
@@ -8,13 +8,18 @@
 #include <QDateTime>
 #include <variant>
 
+#pragma GCC diagnostic ignored "-Wswitch"
```

Comment:
should never ignore warnings :) 
I can accept this :-1:

Reply:
you get this warning either because there is no default in a switch
(really bad) or an enumerator is out of the range ( also bad)

Reply:
Thanks for noticing it. It was only suppressed temporarily until I found the issue. Its fixed now.